app/views/ontologies/sections/metadata/_ontology_description_section.html.haml

```diff
@@ -52,6 +52,7 @@
     - domains = Array(@ontology&.hasDomain) + Array(@submission_latest&.hasDomain)
     - unless domains.empty?
       - l.row do
+        - domains = fix_array_format(domains)
```

Comment:
don't create a helper `fix_array_format` or if needed create a helper `ontology_domaines`

Reply:
I noticed that some ontologies metadata is written this way:
`array = ["value1,value2,value3"]`
instead of:
`array = ["value1","value2","value3"]`
and we need this function to fix the format of these arrays